public/app/core/components/AppChrome/MegaMenu/utils.ts

```diff
@@ -61,8 +61,8 @@ export const enrichWithInteractionTracking = (
     reportInteraction('grafana_navigation_item_clicked', {
       path: newItem.url ?? newItem.id,
       menuIsDocked: megaMenuDockedState,
-      itemIsBookmarked: Boolean(config.featureToggles.pinNavItems && newItem?.parentItem?.id === 'bookmarks'),
-      bookmarkToggleOn: Boolean(config.featureToggles.pinNavItems),
+      itemIsBookmarked: newItem?.parentItem?.id === 'bookmarks',
+      bookmarkToggleOn: true,
```

Comment:
Since you are removing the feature toggles the feature will be always available. That's why I wonder whether it makes sense to keep this key-value-pair in the payload as it's not providing any insights. Would it create a new table in the db if this gets removed? If so and we want to keep the table as is how about setting `bookmarkToggleOn: 'deprecated',`?